frontend_tests/zjsunit/index.js

```diff
@@ -40,6 +40,9 @@ global.write_test_output = output.write_test_output;
 global.append_test_output = output.append_test_output;
 
 var noop = function () {};
+global.clone = function (obj) {
```

Comment:
We tend to just use _.clone() in most of our codebase.  The underscore library is a pretty simple dependency that many of the node tests already use.